vectorizers/_vectorizers.py

```diff
@@ -18,7 +20,7 @@
 from sklearn.neighbors import KernelDensity
 from sklearn.preprocessing import normalize
 
-from collections import defaultdict
+from collections import defaultdict, Counter
```

Comment:
Counter isn't required here
```suggestion
from collections import defaultdict
```